module/x/gravity/keeper/attestation.go

```diff
@@ -306,7 +312,7 @@ func (k Keeper) GetLastObservedEventNonce(ctx sdk.Context) uint64 {
 // the store
 func (k Keeper) GetLastObservedEthereumBlockHeight(ctx sdk.Context) types.LastObservedEthereumBlockHeight {
 	store := ctx.KVStore(k.storeKey)
-	bytes := store.Get([]byte(types.LastObservedEthereumBlockHeightKey))
+	bytes := store.Get(types.LastObservedEthereumBlockHeightKey)
```

Comment:
what lead to this change?

Reply:
I got a suggestion while working on the file that it was a redundant type conversion, since we are now using hashed bytes